tests/transaction_hooks/tests.py

```diff
@@ -32,6 +32,15 @@ def do(self, num):
         Thing.objects.create(num=num)
         transaction.on_commit(lambda: self.notify(num))
 
+    def do_robust(self, num):
+        """Create a Thing instance and notify about it."""
+        Thing.objects.create(num=num)
+
+        def func():
+            self.notify(num)
+
+        transaction.on_commit_robust(func)
```

Comment:
Subjective opinion: keep the same layout as `do`
=>
```suggestion
        Thing.objects.create(num=num)
        transaction.on_commit_robust(lambda: self.notify(num))
```

Reply:
Noted 👍